Python/ceval.c

```diff
@@ -1852,27 +1862,33 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             goto error;
         }
 
-        TARGET(RETURN_VALUE) {
```

Comment:
Reminder that you'll get a merge conflict here -- instruction changes now need to be in bytecodes.c and you need to run `make regen-cases`.